MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include requirements.txt
+include napalm_skeleton/templates/*.j2
```

Comment:
As you don't have any templates defined yet, you can remove lines 2 & 3. But when you'll need, make sure to include using ```include naplam_panos/templates/*.j2``` etc.